lib/Doctrine/Migrations/Metadata/AvailableMigrationsList.php

```diff
@@ -11,6 +11,10 @@
 use function array_values;
 use function count;
 
+/**
+ * The class AvailableMigrationsList represents a sorted list of available migrations.
+ * The migrations in this list might be already executed or not.
```

Comment:
Let's cut to the chase here too
```suggestion
 *  Represents a sorted list of migrations that may or maybe not be already executed.
```